server_installation/topics/operator/external-database.adoc

```diff
@@ -36,6 +37,21 @@ The other properties work in the same way for a hosted or external database. Set
 * `POSTGRES_USERNAME` - Database username
 * `POSTGRES_PASSWORD` - Database password
 * `POSTGRES_SUPERUSER` - Indicates, whether backups should run as super user. Typically `true`.
+* `SSL_MODE` - Indicates, whether to use TLS on the connection to the external PostgreSQL database, check the possible https://www.postgresql.org/docs/current/libpq-ssl.html[values]
```

Comment:
Remove the comma after "Indicates"
Also, what is the subject of the clause "check the possible..." Is that a second action of this property?

Reply:
probably could make it clearer : 

`Indicates whether to use TLS on the connection to the external PostgreSQL database. Please check the possible https://www.postgresql.org/docs/current/libpq-ssl.html[values]`

Reply:
Checking it again, the previous text for POSTGRES_SUPERUSER is using a comma after the Indicates, should I remove it too ? @andymunro